psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -711,13 +711,15 @@ def scf_print_energies(self):
     if self.functional().needs_xc():
         self.set_variable("DFT XC ENERGY", exc)  # P::e SCF
         self.set_variable("DFT VV10 ENERGY", evv10)  # P::e SCF
-        self.set_variable("DFT FUNCTIONAL TOTAL ENERGY", hf_energy + exc + evv10)  # P::e SCF
+        self.set_variable("DFT FUNCTIONAL TOTAL ENERGY", hf_energy + exc)  # P::e SCF
```

Comment:
I don't use DFT enough to understand the significance of this change, but this PR is not the place to change the definition of psivars. I'd make an exception for a genuine bug, but this is just more definition wrangling.

Reply:
VV10 is a weird one. It's like -D in that parameters are set with the EmpiricalDispersion tech. It's like a proper functional piece in where it's actually computed. I think I favor it being a part of E_fctl as it is rather than E_-D, but I couldn't have answered which it was in at present without looking it up, so I could be persuaded otherwise. But definitions would have to change: https://psicode.org/psi4manual/master/glossary_psivariables.html#psivar-DFT-VV10-ENERGY

Reply:
It is definitely part of the functional total energy.

Reply:
No problem, I'll revert the changes and find a workaround for the DFT code. I thought we had agreed on this change in March, but I must've misremembered.